.semaphore/semaphore.yml

```diff
@@ -12,10 +12,6 @@ blocks:
           commands:
             - checkout
 promotions:
-  - name: 02. Sandbox - Update config
```

Comment:
An ansible role exists for sandbox environment in file `simple.org-sandbox`. Should we remove it?

Reply:
Also, a folder of ssh keys exists for sandbox. Can be removed too?